### DISCUSSION TITLE
[deepin 20.1] Image Viewer - AVIF support

### DISCUSSION THREAD

**Discussion by lxwndr** [linuxdeepin/developer-center#2840] [Features Request & Ideas | 特性请求 & 头脑风暴]
Offers compatibility with the AV1 image file format (.avif), which is being adopted by all web browsers.

This format is considered superior to JPEG, PNG, WebP and GIF; to significantly reduce the file size and maintain quality.

It is important to note that it is an open source and royalty-free image format!

deepin-image-viewer 5.6.3.73-1
deepin 20.1 (1030) / OsBuild = 11038.003

**Comment by justforlxz** [comment]
I hope to support heif（

**Comment by justforlxz** [comment]
@xmuli https://github.com/jakar/qt-heif-image-plugin 

please check and add it

**Comment by XMuli** [comment]
ok, I'm going to push it into deepin.

**Comment by zoze0** [comment]
gThumb支持avif马上都两年了。 deepin落后也不至于要这么落后吧

**Reply by BLumia** [reply]
装一下 kimageformats 就支持了，一个 Qt 插件的事...

**Reply by zoze0** [reply]
那啥时候预装啊， 不只是看图软件能打开哦， 文件管理器里格式支持，小图预览。 现在识别不了这种格式

**Reply by BLumia** [reply]
v20 的仓库恐怕不会随便更新包了（虽说如果依赖不多的话还是可能的，但 kimageformats 可能需要点额外的工作才能把 avif 插件移植到 v20）。

可以期待下后续的 v23 仓库，应该是直接有的。

**Comment by zoze0** [comment]
[https://www.bilibili.com/read/cv22543150](https://www.bilibili.com/read/cv22543150)
B站首页已经转为AVIF格式图片了

**Reply by BLumia** [reply]
https://github.com/deepin-community/kimageformats  v23 仓库是有这个包的。
